mirgecom/diffusion.py

```diff
@@ -352,6 +392,8 @@ def diffusion_operator(discr, *args, return_grad_u=False, **kwargs):
     quadrature_tag:
         quadrature tag indicating which discretization in *discr* to use for
         overintegration
+    volume_dd: grudge.dof_desc.DOFDesc
+        the DOF descriptor of the volume on which to apply the operator
```

Comment:
By this point, I've seen `volume_dd` and `dd_vol`. Which of the two is it going to be? :)

Reply:
Fixed.